### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/bd0f50a1c0a7455115cc2bf9eb3ae9027586a024

### CODE REVIEW

**Comment by alexgibson** [comment]
@peiying2 there are a couple of small string tweaks coming, please hold on merging and I'll push another update shortly.

**Comment by alexgibson** [comment]
@peiying2 ok updates made, this is ready for Pontoon. Thanks

File: en/products/vpn/landing.ftl [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+vpn-landing-brand-trust-heading = A VPN from a brand you can trust
+
+# Variables:
+#   $url (url) - https://www.mozilla.org/about/manifesto/
+vpn-landing-brand-trust-desc = For more than 20 years, { -brand-name-mozilla } has a track record of putting people first and fighting for online privacy. Backed by a non-profit, we are committed to building a better and healthier internet for all people. Everything we make is part of our mission and follows our <a href="{ $url }">principles</a>.
+
+## FAQ section
+
+vpn-landing-faq-heading = FAQs
+vpn-langing-faq-uses-question-heading = What is a VPN and its uses?
```
Comment (peiying2) [inline en/products/vpn/landing.ftl:36]: Should this be What is a VPN and what are the uses?

File: en/products/vpn/shared.ftl [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/vpn/
+
+vpn-shared-product-name = { -brand-name-mozilla-vpn }
+vpn-shared-subscribe-link = Get { -brand-name-mozilla-vpn }
+vpn-shared-waitlist-link = Join the Waitlist
+vpn-shared-sign-in-link = Already a subscriber?
+vpn-shared-available-countries = We currently offer { -brand-name-mozilla-vpn } in the US, the UK, Canada, New Zealand, Singapore and Malaysia.
```
Comment (peiying2) [inline en/products/vpn/shared.ftl:11]: The list of countries - the style is not consistent. Either add a comma to all three before `and CountryName` or remove it.

**Review by peiying2 — COMMENTED** [review]
@alexgibson please check my comments and see if they are valid.

**Review by peiying2 — APPROVED** [review]
LGTM. Thanks for the updates.

**Pull request merged**